code/game/objects/items/devices/von-krabin.dm

```diff
@@ -0,0 +1,71 @@
+/obj/item/device/von_krabin
+	name = "Von-Krabin Stimulator"
+	desc = "The Von-Krabin Stimulator."
```

Comment:
Give a brief description of what it does at least.

Reply:
Fixed.